integrations/reporting_tools/prettytable_object.py

```diff
@@ -0,0 +1,42 @@
+"""
```

Comment:
@shivahari @sravantit25 I feel good to move this file to core helper. What do you think???

Reply:
> @shivahari @sravantit25 I feel good to move this file to core helper. What do you think???

This is a custom plugin for `pytest` that overwrites a default `TerminalWriter` object that generally prints the `Failure Summary`. I was thinking this can be considered an integration rather than a core helper since.